client/src/components/Posts.js

```diff
@@ -0,0 +1,36 @@
+import Post from "./Post"
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const Posts = () => {
+  const [posts, setPosts] = useState([]);
+
+  const fetchPosts = async () => {
+    const res = await axios.get("https://laravel-crud-practice.herokuapp.com/post");
+    if (res.status === 200) {
+      const postData = res.data.posts.reverse();
+      setPosts(postData);
+    }
+  }
+
+  useEffect(() => {
+    fetchPosts();
+  }, []);
+
+  const deletePost = async (id) => {
+    const res = await axios.delete(`https://laravel-crud-practice.herokuapp.com/post/${id}`);
+    if (res.data.status === 200) {
+      fetchPosts();
```

Comment:
on deleting you don't need to fetchPosts again, you can just delete it from state if (res.status===200).
Also it is preferred that if the status code is not 200, we need to display the error occurred on frontend, for better UX and console.logs for better debugging

Reply:
- Yeah, I figured that out but the CRUD operations are performed without authentication which means the system and data is decentralised. So, in that case, it's better to fetch all the data again after any of the CRUD operations. 
- Maybe in future, I will add authentication but as of now, I will regard this as my final build. 
- I have a development branch locally for debugging. That is why I don't log anything in this production branch.
- About next weekend I will start my work on some better error handling strategies.